src/test/java/com/crud/tasks/trello/facade/TrelloFacadeTestSuite.java

```diff
@@ -24,84 +22,76 @@
     @Test
     public void testMapToBoard() {
         //Given
-        List<TrelloListDto> trelloList = new ArrayList<>();
-        trelloList.add(new TrelloListDto("1", "trello list dto", false));
-
-        TrelloBoardDto trelloBoard = new TrelloBoardDto("1", "board_dto", trelloList);
-        List<TrelloBoardDto> boardDtoList = new ArrayList<>();
-        boardDtoList.add(trelloBoard);
+        List<TrelloListDto> trelloList = Arrays.asList(new TrelloListDto("1", "trello list dto", false));
+        List<TrelloBoardDto> boardDtoList = Arrays.asList(new TrelloBoardDto("1", "board_dto", trelloList));
 
         //When
         List<TrelloBoard> mappedList = trelloMapper.mapToBoard(boardDtoList);
 
         //Then
-        assertEquals(boardDtoList.get(0).getName(), mappedList.get(0).getName());
-        assertEquals(boardDtoList.get(0).getId(), mappedList.get(0).getId());
-        assertEquals(boardDtoList.get(0).getLists().get(0).isClosed(), mappedList.get(0).getLists().get(0).isClosed());
+        assertThat(mappedList)
+                .extracting(e -> e.getId(), e -> e.getName())
+                .contains(tuple("1", "board_dto"));
+        assertThat(mappedList.contains(trelloList)).isEqualTo(boardDtoList.contains(trelloList));
```

Comment:
nie rozumiem tej asercji

Reply:
Chcę porównać dwie listy, i tylko to przyszło mi do głowy po innych próbach..